docs/PC-8800.md

```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
 | N88EXT2.ROM | B7C8BCEA219B77D9CC3EE0EFAFE343CC307425D1 | AF2B6EFA | 8 KB |
 | N88EXT3.ROM | EFCE0B51CAB9F0DA6CF68507757F1245A2867A72 | 7713C519 | 8 KB |
 
-If you add these 2 files too, you will have the translation of the menus in kanji characters:
+Adding the next 2 files :
+
+Menus (Quasi88 only) and Japanese games will display the Japanese kanji characters. [Quasi88, lr-quasi88]
+
+Not adding the next 2 files :
+
+Menus will be in English (Quasi88 only) and Japanese games will **not** display the Japanese kanji characters. [Quasi88, lr-quasi88]
```

Comment:
This section is superfluous, just specify what the BIOS files are for - allow the display of Japanese Kanji characters in the emulator.